cli/bench/utils.go

```diff
@@ -0,0 +1,13 @@
+package bench
+
+import "math/rand"
+
+var letterRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+func RandStringRunes(n int) string {
+	b := make([]rune, n)
+	for i := range b {
+		b[i] = letterRunes[rand.Intn(len(letterRunes))]
+	}
+	return string(b)
+}
```

Comment:
I think this function should be placed here: https://github.com/tarantool/cartridge-cli/blob/master/cli/common/utils.go

Reply:
The function was removed cause I found the same in common/utils